lib/ansible/utils/unsafe_proxy.py

```diff
@@ -53,28 +53,33 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-from ansible.module_utils.six import string_types, text_type, binary_type
 from ansible.module_utils._text import to_text
 from ansible.module_utils.common._collections_compat import Mapping, MutableSequence, Set
+from ansible.module_utils.six import string_types, binary_type, text_type
 
 
-__all__ = ['UnsafeProxy', 'AnsibleUnsafe', 'wrap_var']
+__all__ = ['AnsibleUnsafe', 'wrap_var']
 
 
 class AnsibleUnsafe(object):
     __UNSAFE__ = True
 
 
-class AnsibleUnsafeText(text_type, AnsibleUnsafe):
+class AnsibleUnsafeBytes(binary_type, AnsibleUnsafe):
     pass
 
 
-class AnsibleUnsafeBytes(binary_type, AnsibleUnsafe):
+class AnsibleUnsafeText(text_type, AnsibleUnsafe):
     pass
 
 
 class UnsafeProxy(object):
     def __new__(cls, obj, *args, **kwargs):
```

Comment:
I cannot comment directly on the line, since it wasn't changed in this PR.  But can we short circuit with a check like the following before line 87:

```
if isinstance(AnsibleUnsafe):
    return obj
```

Reply:
jinx!

Reply:
I did not think of changing `UnsafeProxy`. Done.